examples/deselection.rs

```diff
@@ -23,34 +26,34 @@ fn setup(
 ) {
     // plane
     commands
-        .spawn_bundle(PbrBundle {
+        .spawn(PbrBundle {
             mesh: meshes.add(Mesh::from(shape::Plane { size: 5.0 })),
             material: materials.add(Color::rgb(0.3, 0.5, 0.3).into()),
             ..Default::default()
         })
-        .insert_bundle(PickableBundle::default());
+        .insert(PickableBundle::default());
```

Comment:
consolidating as many operations as possible to improve ergonomics and cut down on archetype moves is also recommended now, ex:

```rust
commands.spawn((
        PbrBundle {
            mesh: meshes.add(Mesh::from(shape::Plane { size: 5.0 })),
            material: materials.add(Color::rgb(0.3, 0.5, 0.3).into()),
            ..Default::default()
        },
        PickableBundle::default(),
    ));
```